src/lib/konnectors.js

```diff
@@ -32,11 +32,22 @@ export function findBySlug (cozy, slug) {
     .then(list => list.length ? list[0] : null)
 }
 
-export function install (cozy, slug, source, timeout = 120000) {
-  if (!slug) throw new Error('Missing `slug` paramater for konnector')
-  if (!source) throw new Error('Missing `source` parameter for konnector')
+export function install (cozy, konnector, timeout = 120000) {
```

Comment:
This default timeout value (also found in src/lib/DataConnectStore.js#L8) should be set in a config file.